scripts/npm-publish.sh

```
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 project_dir=$(cd $(dirname $0); cd ../; pwd)
+flags=$@
```

Shellcheck (0.11.0)

[warning] Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.

(SC2124)